packages/not-next/src/hooks/usePublicClient.ts

```diff
@@ -1,13 +1,13 @@
 import { usePublicClient as usePC } from "wagmi";
 import * as chains from "wagmi/chains";
-import process from "node:process";
+import { type PublicClient } from "viem";
 
 export function usePublicClient(
-  id = chains[process.env.NEXT_PUBLIC_CHAIN_NAME as keyof typeof chains].id,
-) {
+  id = chains[import.meta.env?.VITE_CHAIN_NAME as keyof typeof chains]?.id ?? 1,
+): { shopPublicClient: PublicClient } {
   const shopPublicClient = usePC({
     chainId: id,
-  });
+  }) as PublicClient;
```

Comment:
`as PublicClient;` is probably unessicary. usePC should return the type `PublicClient | Undefined`